.github/workflows/sphinx.yml

```diff
@@ -35,11 +36,10 @@ jobs:
       - name: Build Documentation
         run: make docs
       - name: Extract Source Strings
-        if: github.ref == 'refs/heads/master'
```

Comment:
It's ok to change the cloned repo, we will check that everything works in the PRs.
We will only push to master when already in master.